.rhdh/docker/Dockerfile

```diff
@@ -135,7 +135,9 @@ RUN $YARN build --filter=backend && \
   # Already imported the packages above; need to `yarn install` on the `dist-dynamic` sub-folder for backend plugins
   $YARN --cwd ./dynamic-plugins/imports install-dynamic && \
   $YARN export-dynamic -- --filter=./dynamic-plugins/wrappers/* && \
-  $YARN copy-dynamic-plugins dist
+  $YARN copy-dynamic-plugins dist && \
+  # insert build time into the build-metadata.json
+  now=$(date -u +%FT%TZ); sed -i packages/app/src/build-metadata.json -r -e "s/\"(Last Commit: )(.+)\"/\"\1\2\", \"Build Time: $now\""
```

Comment:
why don't you just run the `versions:metadata` script here?

Reply:
I'm adding `Build Time:` not changing `Last Commit`. For this is the BUILD time, not the last commit time. :D